lib/gui/components/drive-info-modal/templates/drive-info-modal.tpl.html

```diff
@@ -0,0 +1,59 @@
+<div class="modal-header">
+  <h4 class="modal-title">
+    <span class="glyphicon glyphicon-{{ ::modal.options.icon }}"></span>
+    <span>{{ ::modal.options.title }}</span>
+  </h4>
+  <button class="close" ng-click="modal.reject()">&times;</button>
+</div>
+
+<div class="modal-body">
+  <div class="modal-text"
+    ng-if="modal.options.header && modal.options.description">
+    <h4>{{ ::modal.options.header }}</h4>
+    <p>{{ ::modal.options.description }}</p>
+  </div>
+  <div class="modal-selected-drive">
+    <div class="modal-selected-drive-description">
+      {{ ::modal.selection.getDrive().description }}
+
+    </div>
+    <div class="modal-selected-drive-name">
+      {{ ::modal.selection.getDrive().name }} - {{ ::modal.selection.getDrive().size | gigabyte | number:1 }} GB
+
+
+      <span class="label label-warning"
+          ng-show="modal.selection.isDriveLargeEnough(modal.selection.getDrive()) && !modal.selection.isDriveLocked(modal.selection.getDrive()) && !modal.selection.isDriveSizeRecommended(modal.selection.getDrive())">
+          <i class="glyphicon glyphicon-warning-sign"></i>
+          NOT RECOMMENDED</span>
+
+      <span class="label label-danger"
+        ng-hide="modal.selection.isDriveLargeEnough(modal.selection.getDrive())">
+          <i class="glyphicon glyphicon-resize-small"></i>
+          TOO SMALL FOR IMAGE</span>
+
+      <span class="label label-danger"
+        ng-show="modal.selection.isDriveLocked(modal.selection.getDrive()) && modal.selection.isDriveLargeEnough(modal.selection.getDrive())">
```

Comment:
...and the logic on this line is shared between `drive-selector-modal.tpl.html` and `drive-info-modal.tpl.html`, perhaps this logic of categorising drives into "OKAY" (no warning label), "NOT RECOMMENDED", "TOO SMALL FOR IMAGE", "LOCKED" and "SYSTEM DRIVE" ought to be refactored out into the model itself (or some other shared JS file), rather than having the logic embedded inside HTML templates?